source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/Building.java

```diff
@@ -53,4 +57,15 @@ public void setHeight(final double height) {
     this.height = height;
   }
 
+  public double getRadius() {
```

Comment:
Probably need to add this (and related changes) to IMAER as well. Not saying it should be done in this PR, just an observation.

Reply:
Yes this needs to be added.